styles/_tutorials.scss

```diff
@@ -0,0 +1,169 @@
+/* Tutorials ---------------------------------------------------------------- */
+.tutorial-content {
+  #beginner-cards {
+    .card:nth-child(1) .card-footer {
+      background-image: url(../img/cards/4col-magenta-3.svg);
+    }
+  }
+
+  #sdk-cards {
+    .card:nth-child(1) .card-footer {
```

Comment:
> Curious about why different sections have different repeating colors. I wonder if all cards should use the same color order. Would make it easier if more sections get added.

There isn't any specific reason other than having random variety of colors here just makes the page less boring. We don't have that many cards at the moment and having the same color start each section seems monotonous. Since each card is a different thing and we're not mapping each other to a specific topic, might as well keep it fluid.

Reply:
I was thinking maybe a mixin would make sense where we give it an offset of which color to start a section with and then the order goes the same.

Reply:
mixin with an offset sounds like a good idea! Will implement it